json_encoder_test.go

```diff
@@ -114,8 +115,12 @@ func TestJSONEncoderFields(t *testing.T) {
 		{"bool", `"k\\":true`, func(e Encoder) { e.AddBool(`k\`, true) }},
 		{"int", `"k":42`, func(e Encoder) { e.AddInt("k", 42) }},
 		{"int", `"k\\":42`, func(e Encoder) { e.AddInt(`k\`, 42) }},
-		{"int64", `"k":42`, func(e Encoder) { e.AddInt64("k", 42) }},
-		{"int64", `"k\\":42`, func(e Encoder) { e.AddInt64(`k\`, 42) }},
+		{"int64", fmt.Sprintf(`"k":%d`, math.MaxInt64), func(e Encoder) { e.AddInt64("k", math.MaxInt64) }},
```

Comment:
Thanks for updating these tests, can you also add `MinInt64` to the `int64` test?